### DISCUSSION TITLE
question-I look for some contact with 1of11

### DISCUSSION THREAD

**Discussion by pawol** [egzumer/uv-k5-firmware-custom#144] [General]
I look for some contact with 1of11 because he has serious bug with power consumption in his impementation if FW. Ar the moment he has no "issues" tab.

**Comment by egzumer** [comment]
I have no contact with her

**Comment by pawol** [comment]
That is the reason I abandoned that project. One is heavily bugged now.

**Comment by ETO-bigbang** [comment]
> That is the reason I abandoned that project. One is heavily bugged now.

You can leave a comment, and at the bottom of each submission, he may see it.
